server/commands/userProvisioner.ts

```diff
@@ -236,9 +237,8 @@ export default async function userProvisioner({
         isViewer: isAdmin === true ? false : defaultUserRole === "viewer",
         teamId,
         avatarUrl,
-        service: null,
         authentications: authentication ? [authentication] : [],
-      },
+      } as Partial<InferCreationAttributes<User>>,
```

Comment:
Again 😭

Reply:
I can live with it for all the other improvements here